src/99-appendices/01-contributors.md

```diff
@@ -132,6 +132,7 @@ your name is not listed, please add it.
 -   Pradeep Reddy Raamana 💻🔧
 -   Vasudev Raguram 💻🎨📖🔧
 -   Nick F. Ramsey 📖
+-   Alex Rockhill
```

Comment:
Based on this and MNE-BIDS, it seems like the following codes are applicable:

```suggestion
-   Alex Rockhill 📖🔧
```

Feel free to add others that you feel apply.